src/code_800AF9B0.c

```diff
@@ -2132,11 +2132,11 @@ void func_800B29D8(void *arg0, u16 arg1) {
                     sp1C = phi_v1;
                     func_800B44AC(temp_a0_2);
                     play_sound2(0x49008000);
-                    phi_t0_2 = &D_8018EDED;
+                    phi_t0_2 = &gGameMenuSelectionDepth;
                 }
             }
             if ((phi_v1 & 0x4000) != 0) {
-                *phi_t0_2 = 3;
+                *phi_t0_2 = PLAYER_AMOUNT_SELECTION;
```

Comment:
`ACTIVE_PLAYERS_SELECTION`?
`NUM_PLAYERS`? `PLAYERS_SELECTED`?

I would replace amount with `number`.

Reply:
I think `NUM_PLAYERS` and `PLAYERS_SELECTED` may confuse this with the actual chosen number of players. This is meant to signify the player's location in the main menu is at the point where they are choosing 1P, 2P, 3P, or 4P. 

`PLAYER_NUMBER_SELECTION`?

Reply:
Yea, player num selection is fine.